```
=========================== short test summary info ============================
FAILED tests/autocompletion/test_file_indexer.py::test_updates_index_on_file_creation - assert False
 +  where False = _wait_for(<function test_updates_index_on_file_creation.<locals>.<lambda> at 0x1088262a0>)
FAILED tests/autocompletion/test_file_indexer.py::test_updates_index_on_file_rename - assert False
 +  where False = _wait_for(<function test_updates_index_on_file_rename.<locals>.<lambda> at 0x1184262a0>)
FAILED tests/autocompletion/test_file_indexer.py::test_updates_index_on_file_deletion - assert False
 +  where False = _wait_for(<function test_updates_index_on_file_deletion.<locals>.<lambda> at 0x113874670>)
FAILED tests/autocompletion/test_file_indexer.py::test_updates_index_on_folder_rename - assert False
 +  where False = _wait_for(<function test_updates_index_on_folder_rename.<locals>.<lambda> at 0x11870c5c0>)
FAILED tests/autocompletion/test_file_indexer.py::test_updates_index_incrementally_by_default - assert False
 +  where False = _wait_for(<function test_updates_index_incrementally_by_default.<locals>.<lambda> at 0x113874ca0>)
FAILED tests/autocompletion/test_file_indexer.py::test_rebuilds_index_when_mass_change_threshold_is_exceeded - assert False
 +  where False = _wait_for(<function test_rebuilds_index_when_mass_change_threshold_is_exceeded.<locals>.<lambda> at 0x11870cd50>)
FAILED tests/autocompletion/test_file_indexer.py::test_watcher_toggle_flow_off_on_off - assert False
 +  where False = _wait_for(<function _assert_created_file_is_indexed.<locals>.<lambda> at 0x113875640>)
FAILED tests/onboarding/test_ui_onboarding.py::test_ui_delays_browser_sign_in_url_help - AssertionError: assert 'If your browser did not open, copy this URL (press c).' == ''

  + If your browser did not open, copy this URL (press c).
===== 8 failed, 8487 passed, 15 skipped, 39 warnings in 360.15s (0:06:00) ======
```

You were probably slightly too aggressive with test disabling ^^

--- a/pkgs/by-name/mi/mistral-vibe/package.nix
+++ b/pkgs/by-name/mi/mistral-vibe/package.nix
@@ -13,7 +13,7 @@
 
 python3Packages.buildPythonApplication (finalAttrs: {
   pname = "mistral-vibe";
-  version = "2.23.3";
+  version = "2.24.0";
   pyproject = true;
   __structuredAttrs = true;
 
@@ -21,7 +21,7 @@ python3Packages.buildPythonApplication (finalAttrs: {
     owner = "mistralai";
     repo = "mistral-vibe";
     tag = "v${finalAttrs.version}";
-    hash = "sha256-VvrLcotRl3nyjb0c1sipDMSEpbIGWF2T0rCA0adgSww=";
+    hash = "sha256-Ux5W+9mtvAdlINQfr58Fft4OtpqjoYQrQe3XZwhvovc=";
   };
 
   build-system = with python3Packages; [
@@ -165,47 +165,6 @@ python3Packages.buildPythonApplication (finalAttrs: {
 
     # TypeError: cannot pickle 'itertools.count' object (Python 3.14 compatibility)
     "test_orchestrator_deepcopies_and_stays_functional"
-
-    # Fail in the sandbox
-    # vibe.core.audio_recorder.audio_recorder_port.NoAudioInputDeviceError: No audio input device available
-    "test_audio_stream_yields_chunks"
-    "test_auto_stops_after_max_duration"
-    "test_buffer_mode_audio_stream_yields_nothing"
-    "test_callback_feeds_audio_data"
-    "test_callback_pads_silence_at_end"
-    "test_can_play_multiple_times"
-    "test_can_record_multiple_times"
-    "test_cancel_discards_audio"
-    "test_creates_stream_with_correct_params"
-    "test_finished_callback_resets_state"
-    "test_manual_stop_prevents_on_expire"
-    "test_on_expire_receives_audio"
-    "test_on_finished_called_after_natural_completion"
-    "test_peak_clamps_to_one"
-    "test_peak_updates_from_callback"
-    "test_play_sets_playing_state"
-    "test_play_when_already_playing_raises"
-    "test_silent_audio_has_zero_peak"
-    "test_start_sets_recording_state"
-    "test_start_when_already_recording_raises"
-    "test_stop_closes_stream"
-    "test_stop_from_event_loop_does_not_block"
-    "test_stop_returns_empty_data_in_stream_mode"
-    "test_stop_returns_positive_duration"
-    "test_stop_returns_valid_wav"
-    "test_stop_triggers_on_finished_via_callback"
-    "test_stop_without_drain_returns_promptly"
-    "test_stream_audio_does_not_leak_into_buffer_recording"
-    "test_unsupported_format_raises"
-
-    # AssertionError: assert True is False
-    #  +  where True = VibeApp(title='VibeApp', ...)._rewind_mode
-    "test_rewind_confirm_edits_message_and_prefills_input"
-    "test_rewind_option_selection_with_number_keys"
-
-    # AssertionError: assert 3 == 1
-    # +  where 3 = len([UserMessage(classes='user-message'), UserMessage(classes='...
-    "test_rewind_removes_messages_after_selected"
   ]
   ++ lib.optionals stdenv.hostPlatform.isDarwin [
     # AssertionError
@@ -225,8 +184,18 @@ python3Packages.buildPythonApplication (finalAttrs: {
     # All snapshot tests fail with AssertionError
     "tests/snapshots/"
 
-    # Try to invoke `uv run vibe`
-    "tests/e2e/"
+    # These tests invoke uv run and fail to import the packaged pydantic extension.
+    "tests/e2e/agent_loop_characterization/test_resume.py"
+    "tests/e2e/agent_loop_characterization/test_subagents.py"
+    "tests/e2e/agent_loop_characterization/test_tool_execution.py"
+    "tests/e2e/agent_loop_characterization/test_tool_permissions.py"
+    "tests/e2e/agent_loop_characterization/test_user_interaction.py"
+    "tests/e2e/test_cli_tui_fresh_install.py"
+    "tests/e2e/test_cli_tui_hooks.py"
+    "tests/e2e/test_cli_tui_onboarding.py"
+    "tests/e2e/test_cli_tui_session_exit.py"
+    "tests/e2e/test_cli_tui_streaming.py"
+    "tests/e2e/test_cli_tui_tool_approval.py"
 
     # ACP tests require network access
     "tests/acp/test_acp_entrypoint_smoke.py"

